xpublish/rest.py

```diff
@@ -258,21 +300,27 @@ def _init_app(self):
 
     @property
     def app(self) -> FastAPI:
-        """Returns the :class:`fastapi.FastAPI` application instance."""
+        """Returns the :class:`fastapi.FastAPI` application instance.
+
+        Notes
+        -----
+        Plugins registered with :meth:`xpublish.Rest.register_plugin` after :meth:`xpublish.Rest.app`
+        is accessed or :meth:`xpublish.Rest.serve` is called once may not take effect.
+        """
         if self._app is None:
             self._app = self._init_app()
         return self._app
 
-    def serve(self, host='0.0.0.0', port=9000, log_level='debug', **kwargs):
+    def serve(self, host: str = '0.0.0.0', port: int = 9000, log_level: str = 'debug', **kwargs):
         """Serve this FastAPI application via :func:`uvicorn.run`.
 
         Parameters
         ----------
-        host : str
+        host :
             Bind socket to this host.
-        port : int
+        port :
             Bind socket to this port.
-        log_level : str
+        log_level :
```

Comment:
FYI The colon after the parameter name can be removed. This is already the case in some parts of the project.